packages/web-components/src/components/cta-block/cta-block.ts

```diff
@@ -141,7 +144,10 @@ class C4DCTABlock extends StableSelectorMixin(C4DContentBlock) {
     });
 
     return html`
-      <div ?hidden="${!this._hasBodyContent()}" class="${classes}">
+      <div
+        ?hidden="${!this._hasBodyContent()}"
+        class="${classes}"
+        part="content">
```

Comment:
Lets go with the suggestion from the class, which would be `body`.

```suggestion
        part="body">
```